.travis.yml

```diff
@@ -4,12 +4,11 @@ php:
   - 7.1
   - 7.2
   - 7.3
-  - 7.4snapshot
+  - 7.4
   - nightly
```

Comment:
Should be `8.0`, rather than `nightly`

Reply:
This is a draft, there's a long journey before your review is requested ^^

Reply:
S'alright, just processed it as it hit my inbox :)

Reply:
This phrase sounded a lot less harsh in my mind, sorry